scripts/buildkite/check-stylish.sh

```diff
@@ -2,6 +2,6 @@
 
 set -euo pipefail
 
-stylish-haskell -i `git ls-files -- 'ouroboros-consensus*/*.hs' | grep -v Setup.hs`
+stylish-haskell -i $(git ls-files -- 'ouroboros-consensus*/*.hs' | grep -v -e 'Setup.hs' -e 'Ouroboros/Consensus/Mempool/TxLimits\.hs')
```

Comment:
Also if this is not already reported, I would report it in the stylish-haskell repository.

Reply:
I opened Issue haskell/stylish-haskell#374